### ISSUE TITLE
Recalculate the MAX when changing fee

### ISSUE THREAD

**Issue by Transisto** [mycelium-com/wallet#339]
Low priority. 

Cannot intelligently  change fee after having selected MAX amount

**Comment by Giszmo** [comment]
Marked it as "bug" because lowering the fee after selecting "max amount" results in dust outputs.
Marked as "enhancement" because the right thing to do would be to lock-in a "send max" mode that allowed to change the fee, updating both fee and amount in parallel.
